src/notifications/teams.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+import logging
+import requests
+from typing import Dict
+import json
+
+from db.models import Site, Victim
+from .source import NotificationSource
+
+class TeamsNotification(NotificationSource):
+
+    def _escape_url(url:str) -> str:
+        url_rep = url.replace("http", "hxxp").replace(".", "[.]")
+        return url_rep
+
+    def _send_message_by_incomingwebhook(url:str,message:str) -> requests.models.Response:
+        headers={"Content-type": "application/json"}
+        params={"text":message}
+        response=TeamsNotification._post(url,headers=headers,params=json.dumps(params))
+        return response
+
+    def _post(url:str, headers:dict = None, params:dict = None) -> requests.models.Response:
+        if headers:
```

Comment:
You don't need to break apart this call depending on if `headers` is not `None`, the default value is `None` and that will be accepted by the `requests` API.